index.js

```diff
@@ -1,5 +1,10 @@
+import tty from 'node:tty';
+
+// TODO: Use a better method when it's added to Node.js (https://github.com/nodejs/node/pull/40240)
+const hasColors = tty.WriteStream.prototype.hasColors();
```

Comment:
@Richienb 

Was there a reason for doing this instead of `process.stdout.hasColors()`?

Reply:
I don't remember.

@sindresorhus

Reply:
I believe the answer is in the TODO comment. `process.stdout.hasColors()` was not available then.

Reply:
Sorry, I missed that.

Seems `process.stdout.hasColors()` has been available. However, they're trying to deprecate it because:
- [it's based on the environment](https://github.com/nodejs/node/blob/ae0cfc95d55f35b76be7341969d6ad36dcc000b7/lib/internal/tty.js#L103-L213) rather than the stream
- it's not available when `process.stdout` is not a proper TTY

In the PR, they're trying to expose a standalone function but is blocked by deprecation strategy for the prototype method.